docs/reference/requests/create_transfers.md

```diff
@@ -215,6 +215,15 @@ transfer.
 The transfer was not created. [`Transfer.timeout`](../transfer.md#timeout) is nonzero, but only
 [pending](../transfer.md#flagspending) transfers have nonzero timeouts.
 
+### `closing_transfer_must_be_pending`
+
+The transfer was not created. [`Transfer.flags.pending`](../transfer.md#flagspending) is not set,
+but it should be `true`.
```

Comment:
> but it should be `true`.

We should always explain _why_. For example, why `pending` should be true. This helps the reader understand the spirit/motivation behind the feature, and empathize with why the design makes sense.

Reply:
At present, we're only stating _must_ but without giving the rational reason behind that _must_. So the reader has no way to understand (at least in the context of reading this here), and yet this understanding is more important than the _must/must not_ because understanding _motivates_.

Reply:
i.e. _must_ is good but _why_ is even better.